.changes/nextrelease/presign_additional_headers.json

```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "type": "enhancement",
+    "category": "SignatureV4",
+    "description": "Add support for pre-signing additional headers"

```

Comment:
Can you add a sentence here reflecting that any additional headers that are added and signed must be supplied when ultimately sending the request?